templates/endpoints/models/embedding_models/BAAI--bge-large-en-v1.5.yaml

```diff
@@ -0,0 +1,33 @@
+deployment_config:
```

Comment:
do we need to update readme about embedding models?

Reply:
updated